addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -271,6 +271,7 @@ snippetsEditor.SnippetsMenu.include({
         const _super = this._super(...arguments);
         if (this.$body[0].ownerDocument !== this.ownerDocument) {
             this.$body.on('click.snippets_menu', '*', this._onClick);
+            this.$el.on('click', this._checkEditorToolbarVisibilityCallback);
```

Comment:
Why is that inside the website override and not in web_editor? Also could not this be done via the `events` property?

Reply:
I've put it here because we should register that listener only when the SnippetsMenu is instantiated on another document than the snippets' one.
As for now we instantiate the SnippetsMenu this way only in website (in mass_mailing the SnippetsMenu is in the same document as the snippets, so the listener is already set), these additional listeners were set in that patch.